src/app/components/ProjectResults/ProjectResults.tsx

```diff
@@ -156,62 +156,74 @@ const ProjectResults = () => {
-                          fontFamily: "Lato",
-                          fontWeight: 400,
-                        }}
-                      >
-                        <span className="text-3xl">{cardDetail?.name}</span>
-                      </div>
-                      <div className="card-image h-full">
+                <div className=" p-0 md:p-8 flex-1 flex-col rounded-lg mx-0 md:mx-24 ">
+                  <div className="flex flex-wrap w-full shadow-xl bg-gray-300 ">
+                    <div
+                      className="card-head p-2 text-white bg-red-800 pl-3 w-full"
+                      style={{
+                        fontFamily: "Lato",
+                        fontWeight: 400,
+                      }}
+                    >
+                      <span className="text-3xl">{cardDetail?.name}</span>
+                    </div>
+                    <div className="break"> </div>
+
+                    <div className="w-full h-2/5 xl:w-3/5 xl:h-4/5 md:w-1/2 ">
+                      <div className="card-image h-full p-3">
                         {cardDetail?.image_url !== "" ? (
                           <img
                             src={cardDetail?.image_url}
-                            className="w-full h-full object-fill"
+                            className="w-full h-full object-fill shadow-md"
                             alt="image"
                           ></img>
                         ) : null}
                       </div>
                     </div>
-                    <div className="w-full md:w-1/2 xl:w-3/4 flex flex-col p-4">
-                      <div
-                        className="text-xl flex justify-between my-2 w-full"
-                        style={{
-                          fontFamily: "Lato",
-                          fontSize: "1.5rem",
-                          lineHeight: "1.75rem",
-                          fontWeight: 400,
-                          wordSpacing: "0.5rem",
-                        }}
-                      >
-                        {cardDetail?.description}
-                        <br />
-                      </div>
-
+                    <div className="overflow-clip overflow-hidden w-full text-sm xl:text-xl md:w-1/2 xl:w-2/5 flex flex-col pb-2 mx-0 text-center justify-center shadow-none">
                       <div
                         style={{
-                          fontFamily: "Lato",
-                          fontSize: "1.5rem",
-                          lineHeight: "1.75rem",
-                          fontWeight: 300,
-                          wordSpacing: "0.5rem",
+                          fontFamily: "'Zen Maru Gothic', sans-serif",
+                          wordSpacing: "0rem",
+                          height: "80%",
+                          width: "80%",
+                          textAlign: "center",
+                          margin: "0 0 0 2.3rem",
+                          overflow: "scroll",
```

Comment:
change overflow:scroll to overflowY: auto, this will add scroll only when necessary.

Reply:
added overflowY: auto

Reply:
remove the overflow: scroll itself